### ISSUE TITLE
[Medium]: Color Contrast: Insufficient color contrast for border of form field.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#8417]
## Action Performed:
1. Using Chrome, open URL: staging.new.expensify.com
2. Navigate and activate the '+ (New)' button.
3. Navigate to the 'New chat' button.
4. Navigate to the "Name, email, or phone number" form field using the Tab key.
5. Observe that the analyser reports that the contrast ratio is not sufficient.

## Expected Result:
The border of the input field should have a contrast ratio against the page background of 3:1 or greater.

## Actual Result:
Background/foreground color combinations with an insufficient contrast ratio was found on the screen. For instance, the #ECECEC border of "Name, email, or phone number" form field has a contrast ratio of 1.2:1 with the #FFFFFF background.

**Other Occurrences:**
Similar issue repro on PR #8173, #8604

## Workaround:
Yes

## Area issue was found in:
New chat

## Failed WCAG checkpoints
1.4.11

## User impact:
When sufficient contrast ratios are not met, users with low vision may have trouble identifying user interface components.

## Suggested resolution:
Ensure that the color contrast for all user interface controls in their default as well as active, hover and focus states is 3:1 with the adjacent content. 

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:**  1.1.49-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Reference link:** https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html
**Issue reported by:** Sumit

![#8141_CCA fails for border of input](https://user-images.githubusercontent.com/98817464/161135957-cbed1a74-c88b-47a5-bc3e-25b93dba7e3d.png)
